### ISSUE TITLE
Size increase between 1.9.46 and new versions

### ISSUE THREAD

**Issue by kouts** [autoNumeric/autoNumeric#469]
There's been a big size increase between the old 1.9.x and new versions of autonumeric.
The minified version of autonumeric 1.9.46 is 20.3 KB whereas the current version is  63.8 KB.
That's three times the size of the old version.
Are there any plans to slim it down or maybe have a "lite" version with less features, say without currencySymbol etc.

**Comment by AlexandreBonneau** [comment]
Indeed, the size has gone up quite a bit, along with the number of [new features](https://github.com/autoNumeric/autoNumeric/blob/next/CHANGELOG.md)!

I would be open to the idea of a *lite* version without the `predefinedOptions` and eventually a trimmed down version of `AutoNumericEnum` (but then that would mean a limited support for exotic non-usual keys).

But to really save some space that would mean removing some features as you said, and that *could* be hard to do.
[Pull requests](https://github.com/autoNumeric/autoNumeric/blob/next/CONTRIBUTING.md) are welcome if anybody find a clean way to create that configurability, without affecting the performance too much.

**Comment by AlexandreBonneau** [comment]
For info, as of today, minified version `4.0.2` found in `dist` is `167 Kio` big, but compressed as a gzip (which any modern browser [should accept](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Encoding)), it drops to `36.7 Kio`.

![gzipped_an](https://user-images.githubusercontent.com/7248877/28939245-5d1d74f6-782c-11e7-9c6c-c18a3728d711.png)
